tests/prefix_caching/test_prefix_caching.py

```diff
@@ -53,12 +55,26 @@ def test_mixed_requests(
             model,
             dtype=dtype,
             enable_prefix_caching=True,
+            block_size=block_size,
     ) as vllm_model:
         # Run the first prompt so the cache is populated
         vllm_outputs = vllm_model.generate_greedy([cached_prompt], max_tokens)
 
         # Run all the promopts
-        vllm_outputs = vllm_model.generate_greedy(example_prompts, max_tokens)
+        greedy_params = SamplingParams(temperature=0.0, max_tokens=max_tokens)
+        req_outputs = vllm_model.model.generate(example_prompts, greedy_params)
```

Comment:
What's the difference between this way and `.generate_greedy`?

Reply:
`generate_greedy()` doesn't expose all the other fields in `RequestOutput`, only tokens and texts.